### DISCUSSION TITLE
[Maintaining large open source projects] Thank you!

### DISCUSSION THREAD

**Discussion by adityaruplaha** [github/india#50] [Students]
A huge thanks to the speakers for their enlightening talk. I am a 17 year old high school student, and recently became a maintainer at a relatively major project, and its honestly empowering. It has changed my life, knowing my actions have real large scale consequences for thousands of users. I could relate so much to what was said. I just want to thank GitHub for arranging this event, and also for helping me grow as a person. Coding is my primary mode of reacreation, and it has really helped me emotionally to be able to show the world what I can create. Thank you. :)

**Comment by FRTMentor** [comment]
GitHub repository is very user friendly easy to learn and understood.
Please visit the GitHub site, try to create an account similar to the FRT.
And as GitHub landing page will guide to create and and ad the repository.
Also please go through some GitHub learning video's or google links before creating.
